examples/src/main/python/hbase_inputformat.py

```diff
@@ -72,6 +75,9 @@
         keyConverter=keyConv,
         valueConverter=valueConv,
         conf=conf)
+    # hbase_rdd is a RDD[dict]
```

Comment:
I am sorry that I misunderstood your meaning. 
I thought that you mean we should change the RDD......